src/App.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import {
+  BrowserRouter,
+  Switch,
+  Route,
+  // @ts-ignore
```

Comment:
You can easily solve these issues by adding `@types/react-router-dom` to the list of dev dependencies for your project

Reply:
Nice, I've just removed the unnecessaries ts-ignores! Thanks :)